ci/setup-common.yml

```diff
@@ -57,9 +57,13 @@ steps:
     env | sort
   displayName: 'Update environment'
 
-- task: DownloadPipelineArtifact@0
+- task: DownloadPipelineArtifact@2
   inputs:
-    pipelineId: $(openorienteering.superbuild.buildId)
+    buildType: 'specific'
+    project: $(System.TeamProjectId)
+    buildVersionToDownload: 'latest'
+#    pipelineId: $(openorienteering.superbuild.buildId)
```

Comment:
This is the part which is left to resolve: Pick a particular build, not the `latest`.
(IIRC that's why I didn't update this task yet.)

Reply:
If I understand you correct, specifying a particular build instead of the latest one ensures that any yet unstable change of superbuild won't break mapper builds. I now pick the pipeline run that you marked by setting its retention to forever. However, if there is a stable update of superbuild and thus a new pipeline run, then Mapper's setup-common.yml would need to be updated.
BTW: How is the variable openorienteering.superbuild.buildId set? Is this always pointing to the latest build?